hardware/blink1/blink1.html

```diff
@@ -1,5 +1,8 @@
-
 <script type="text/x-red" data-template-name="blink1">
+    <div class="form-row">
```

Comment:
Why move this ? usual default is last field as it's optional.

Reply:
I checked multiple default nodes (Inject, Comment, Function) and mostly the name field is the first field in the list even though being optional.